doc/devel/documenting_mpl.rst

```diff
@@ -47,7 +47,7 @@ requirements that are needed to build the documentation. They are listed in
 * IPython
 * numpydoc>=0.4
 * Pillow
-* sphinx-gallery>=0.1.13
+* sphinx-gallery>=0.1.13 (>=0.2.0 on windows)
```

Comment:
I would just document >=0.2.0 everywhere.  The benefit of allowing 0.1.13 on non-Windows is minimal.